ferrocene/doc/specification/src/changelog.rst

```diff
@@ -18,6 +18,37 @@ with the change that has been applied due to it.
    just the language changes that had an impact to the FLS. See the `release
    notes`_ for a full list of changes.
 
+Language changes in Rust 1.86.0
+-------------------------------
+
+- `Stabilize upcasting trait objects to supertraits. <https://github.com/rust-lang/rust/pull/134367>`_
+
+  - New paragraph: :p:`fls_QB4c6FNKxaPl`
+
+- `Allow safe functions to be marked with the #[target_feature] attribute. <https://github.com/rust-lang/rust/pull/134090>`_
+
+  - Changed paragraph: :p:`fls_3qj3jvmtxvx6`
+
+- `The missing_abi lint now warns-by-default. <https://github.com/rust-lang/rust/pull/132397>`_
+
+  - No change: Lints are not part of the FLS
+
+- Rust now lints about double negations, to catch cases that might have intended to be a prefix decrement operator (--x) as written in other languages. This was previously a clippy lint, clippy::double_neg, and is `now available directly in Rust as double_negations. <https://github.com/rust-lang/rust/pull/126604>`_
+
+  - No change: Lints are not part of the FLS
+
+- `More pointers are now detected as definitely not-null based on their alignment in const eval. <https://github.com/rust-lang/rust/pull/133700>`_
+
+  - No change: rules on when pointers are null were not documented
+
+- `Empty repr() attribute applied to invalid items are now correctly rejected. <https://github.com/rust-lang/rust/pull/133925>`_
+
+  - No change: this bug was not documented
```

Comment:
Do we describe that this attribute is not permitted on non ADT items? If not w eshould fix that, if yes, then

```suggestion
  - No change: The FLS correctly described the intended applicability of the attribute
```

Reply:
This term seems to restrict them to ADTs: https://github.com/ferrocene/ferrocene/blob/ae7d05fe8d66b7e3d5bba6e86ee187e921644f56/ferrocene/doc/specification/src/attributes.rst#L1349